src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -143,6 +143,16 @@ warn.validator.concealed.public.class=\
         Warning: entry {0} is a public class\n\
         in a concealed package, placing this jar on the class path will result\n\
         in incompatible public interfaces
+warn.validator.duplicate.cen.entry=\
+        Warning: {0} copies of {1} is detected in central directory
+warn.validator.duplicate.loc.entry=\
+        Warning: {0} copies of {1} is detected in local file header
+warn.validator.cen.only.entry=\
```

Comment:
I would change

`Warning: Entry {0} in central directory is not in local file header`
to
`Warning:  An equivalent local file header was not found  for the central directory header for the file name: {0} `